OneSignalSDK/app/gradlew

```diff
@@ -0,0 +1,172 @@
+#!/usr/bin/env sh
```

Comment:
Were all these Gradle changes made, in order to use the demo app as an independent project?

Reply:
Ya we don't need any of these files under `OneSignalSDK/app`. We actually don't have a "dev" app anymore. The SDK workspace points to the example to share the same code